CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ current changes on git with [previous release tags][git_tag_comparison].
 ### Changed
 - [Breaking changes to timer API][914]
 - [Removed timer auto-ticking system][931]
+- [Breaking changes to Time API][934]
```

Comment:
These links will be broken because we're missing an "index" for the "unchanged" section. Can you add one and fill in the links for the three changelog items?

(see the previous release changelogs for examples)

Reply:
Sure!